structure_tests/tests/cloudbuild.yaml.in

```diff
@@ -0,0 +1,5 @@
+steps:
+        - name: gcr.io/gcp-runtimes/structure-test-test:${TEST_TAG}
+          args: [
+          '--image', 'gcr.io/google-appengine/debian8:latest',
```

Comment:
sure. I would still argue that it makes sense to run tests against the structure test image in addition to some expected set of tests (Debian obviously works fine). these are more "integration" tests against the structure test image itself if that makes sense (running the tests against some other image and verifying they work); I think we should also do a few checks on the image itself, like making sure the test binary is in the root directory, making sure the /test volume is mounted, etc.